components/form-builder/store/useTemplateStore.ts

```diff
@@ -92,7 +92,8 @@ const useTemplateStore = create<ElementStore>()(
         state.form.elements[index].properties.choices = choices;
       });
     },
-    getSchema: () => getSchema(get()),
+    getSchema: () => getSchemaFromState(get()),
+    getSerializedSchema: () => JSON.stringify(getSchemaFromState(get()), null, 2),
```

Comment:
A case could probably be made to combine these into one. Currently we don't call getSchema directly from anywhere.